sdk/core/core/inc/az_http_transport.h

```diff
@@ -85,12 +85,16 @@ AZ_NODISCARD az_result
 az_http_request_get_header(_az_http_request const* request, int32_t index, az_pair* out_header);
 
 /**
- * @brief Get parts of an HTTP request.
+ * @brief Get parts of an HTTP request. NULL in accepted to ignore getting any parts, for example,
+ * call this function like below to get only the http method and ignore getting url and body
+ *   az_http_request_get_parts(request, &method, NULL, NULL)`
```

Comment:
You are missing a back tick in the beginning of this line of code. Can you check how this renders in doxygen?

Also, can you surround NULL in all the places within this APIs docs with back ticks, i.e. `NULL`?